metrics/metrics-dropwizard/src/main/java/org/apache/james/metrics/dropwizard/DropWizardGaugeRegistry.java

```diff
@@ -19,14 +19,19 @@
 
 package org.apache.james.metrics.dropwizard;
 
+import java.util.concurrent.TimeUnit;
+
 import javax.inject.Inject;
 
 import org.apache.james.metrics.api.Gauge;
 import org.apache.james.metrics.api.GaugeRegistry;
 
+import com.codahale.metrics.CachedGauge;
 import com.codahale.metrics.MetricRegistry;
 
 public class DropWizardGaugeRegistry implements GaugeRegistry {
+    private static final int CACHE_EXPIRACY_IN_SECONDS = 10;
```

Comment:
10 seconds seems really too much for me, 100ms should be fine for our purpose, 1s max.